ESACCI/read-biomass-ESACCI.py

```diff
@@ -0,0 +1,255 @@
+import numpy as np
+from netCDF4 import Dataset
+import pylab as plt
+import os
+import sys
+import time
+import datetime
+from mpl_toolkits.basemap import Basemap
+import math
+import glob
+from urllib.request import urlretrieve
+from subroutines import *
+from PIL import Image
+from pylab import *
+from osgeo import gdal, ogr, osr
+
+# set up Data directory 
+DataDir = "/Users/mingquan/newDATA"
```

Comment:
We want this script to be general, it should work on anyone's computer. Let's assume that the users will need to download source data to this directory and the output will also be written to this directory. In that case, we should change this to "./"